skimage/feature/_canny.py

```diff
@@ -60,11 +60,15 @@ def canny(image, sigma=1., low_threshold=None, high_threshold=None, mask=None,
         Greyscale input image to detect edges on; can be of any dtype.
     sigma : float
         Standard deviation of the Gaussian filter.
-    low_threshold : float
+    low_threshold : float or callable
         Lower bound for hysteresis thresholding (linking edges).
+        If callable is given, it is applied to the gradient image
+        to generate the low_threshold.
```

Comment:
Maybe you could extend the Examples section to illustrate this possibility, maybe with percentiles?

Reply:
@jmetz, this docstring statement isn't bad but it's not great either. Specifically, I want it to state the type of the callable: takes in `image` as its only input and returns either a float or an array of the same shape as `image` as output.